### ISSUE TITLE
Adding Accessibility/Aria/A11y to List

### ISSUE THREAD

**Issue by ParkerPosy** [bvaughn/react-virtualized#1296]
Hello!

FEATURE REQUEST

At the company I work for we use your repository for rendering some of our lists. 
You'll see in the link below, in example 3, the listbox very closely resembles an non-virtualized version of your list element EXCEPT, it's accessible. Your list element already allows for almost all of the functionality needed but you'll see they use two more props that you don't have. `aria-labelledby` and `aria-activedescendant`. These two props greatly expand the functionality of your list element and would provide developers with very mutable abilities to add more functionality to their lists. Aria-labelledby takes a string of id(s) to combine together into an aria-label to be read. This is much easier than just using aria-label because rather than using a string for your name twice, you can reference the first one. Aria-activedescendant allows for focus changes to be heard from the screenreader.

TL:DR add `aria-labelledby` and `aria-activedescendant` to List

Thank you!

https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/listbox/listbox.html

**Comment by adirhaz1z** [comment]
Any update on this? :)
